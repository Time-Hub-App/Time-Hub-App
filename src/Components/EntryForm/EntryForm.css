form input[type='submit'],
form select {
  background-color: transparent;
  margin: 10px;
}

form select {
  padding: 8px;
  border-radius: 5px;
  border: #727397 3px solid;
}

input {
  margin: 10px;
}
