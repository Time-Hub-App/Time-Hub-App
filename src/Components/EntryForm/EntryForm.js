import React from 'react';
import EmotionWheel from '../EmotionWheel/EmotionWheel';
import './EntryForm.css';

export default function EntryForm({ emotion, setEmotion, text, setText, formHandler }) {
  return (
    <form onSubmit={formHandler}>
      <label>Entry: </label>
      <input type="text" value={text} onChange={(e) => setText(e.target.value)} />
      <EmotionWheel emotion={emotion} setEmotion={setEmotion} />
      <input type="submit" value="save" />
    </form>
  );
}
