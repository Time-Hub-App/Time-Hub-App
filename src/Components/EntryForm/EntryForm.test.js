import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router';
import EntryForm from './EntryForm';


it('should render entry form', async () => {
  const { container } = render(
    <EntryForm />
  );
  
  expect(container).toMatchSnapshot();
});