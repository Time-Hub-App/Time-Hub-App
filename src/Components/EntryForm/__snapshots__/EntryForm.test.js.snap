// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render entry form 1`] = `
<div>
  <form>
    <label>
      Entry: 
    </label>
    <input
      type="text"
      value=""
    />
    <div>
      <label>
        Emotion: 
      </label>
      <select>
        <option>
          Love
        </option>
        <option>
          Joy
        </option>
        <option>
          Surprise
        </option>
        <option>
          Sadness
        </option>
        <option>
          Anger
        </option>
        <option>
          Fear
        </option>
        <option>
          Peaceful
        </option>
        <option>
          Tenderness
        </option>
        <option>
          Desire
        </option>
        <option>
          Longing
        </option>
        <option>
          Affectionate
        </option>
        <option>
          Enthralled
        </option>
        <option>
          Elation
        </option>
        <option>
          Enthusiastic
        </option>
        <option>
          Optimistic
        </option>
        <option>
          Proud
        </option>
        <option>
          Cheerful
        </option>
        <option>
          Happy
        </option>
        <option>
          Content
        </option>
        <option>
          Moved
        </option>
        <option>
          Overcome
        </option>
        <option>
          Amazed
        </option>
        <option>
          Confused
        </option>
        <option>
          Stunned
        </option>
        <option>
          Despair
        </option>
        <option>
          Neglected
        </option>
        <option>
          Shameful
        </option>
        <option>
          Disappointed
        </option>
        <option>
          Suffering
        </option>
        <option>
          Disgust
        </option>
        <option>
          Envy
        </option>
        <option>
          Irritable
        </option>
        <option>
          Exasperated
        </option>
        <option>
          Rage
        </option>
        <option>
          Horror
        </option>
        <option>
          Nervous
        </option>
        <option>
          Insecure
        </option>
        <option>
          Terror
        </option>
        <option>
          Scared
        </option>
        <option>
          Satisfied
        </option>
        <option>
          Relieved
        </option>
        <option>
          Compassionate
        </option>
        <option>
          Caring
        </option>
        <option>
          Infatuation
        </option>
        <option>
          Passion
        </option>
        <option>
          Attracted
        </option>
        <option>
          Sentimental
        </option>
        <option>
          Fondness
        </option>
        <option>
          Romantic
        </option>
        <option>
          Rapture
        </option>
        <option>
          Enchanted
        </option>
        <option>
          Jubilation
        </option>
        <option>
          Euphoric
        </option>
        <option>
          Zeal
        </option>
        <option>
          Excited
        </option>
        <option>
          Hopeful
        </option>
        <option>
          Eager
        </option>
        <option>
          Illustrious
        </option>
        <option>
          Triumphant
        </option>
        <option>
          Blissful
        </option>
        <option>
          Jovial
        </option>
        <option>
          Delighted
        </option>
        <option>
          Amused
        </option>
        <option>
          Pleased
        </option>
        <option>
          Touched
        </option>
        <option>
          Stimulated
        </option>
        <option>
          Astounded
        </option>
        <option>
          Speechless
        </option>
        <option>
          Awe-struck
        </option>
        <option>
          Astonished
        </option>
        <option>
          Perplexed
        </option>
        <option>
          Disillusioned
        </option>
        <option>
          Shocked
        </option>
        <option>
          Powerless
        </option>
        <option>
          Grief
        </option>
        <option>
          Lonely
        </option>
        <option>
          Isolated
        </option>
        <option>
          Guilty
        </option>
        <option>
          Regretful
        </option>
        <option>
          Displeased
        </option>
        <option>
          Dismayed
        </option>
        <option>
          Sorrow
        </option>
        <option>
          Depressed
        </option>
        <option>
          Hurt
        </option>
        <option>
          Agony
        </option>
        <option>
          Revolted
        </option>
        <option>
          Contempt
        </option>
        <option>
          Jealous
        </option>
        <option>
          Resentful
        </option>
        <option>
          Aggravated
        </option>
        <option>
          Annoyed
        </option>
        <option>
          Frustrated
        </option>
        <option>
          Agitated
        </option>
        <option>
          Hostile
        </option>
        <option>
          Hate
        </option>
        <option>
          Dread
        </option>
        <option>
          Mortified
        </option>
        <option>
          Anxious
        </option>
        <option>
          Worried
        </option>
        <option>
          Inadequate
        </option>
        <option>
          Inferior
        </option>
        <option>
          Hysterical
        </option>
        <option>
          Panic
        </option>
        <option>
          Helpless
        </option>
        <option>
          Frightened
        </option>
      </select>
    </div>
    <input
      type="submit"
      value="save"
    />
  </form>
</div>
`;
